src/features/resources/ResourceItem.tsx

```diff
@@ -53,32 +75,117 @@ const ResourceItem = ({ resource, event }: { resource: Types.Resource; event: an
     updateFavorites();
   };
 
+  const { data, isLoading, isSuccess } = useStatus();
+  const [showDialog, setShowDialog] = useState(false);
+  const [itSystemError, setItSystemError] = useState(false);
+  const [systemCheckedAt, setSystemCheckedAt] = useState(new Date());
+  const cancelRef = useRef<HTMLButtonElement>(null);
+  const open = () => setShowDialog(true);
+  const close = () => setShowDialog(false);
+
+  useEffect(() => {
+    if (isSuccess) {
+      // look for the resource's corresponding IT system
+      if (resource.hasOwnProperty('itSystem') && data !== undefined) {
+        var result = data.find((system) => system.name === resource.itSystem);
+
+        if (resource.itSystem && result !== undefined && result.status !== 1) {
+          setItSystemError(true);
+        }
+        // update date checked time
+        setSystemCheckedAt(new Date());
+      }
+    }
+  }, [isSuccess]);
+
   return (
-    <ListItemFlex>
-      <ListItemResourceLink
-        href={resource.link}
-        target="_blank"
-        onClick={() => {
-          event();
-          if (!resource.excludeTrending) {
-            TrendingEvent(resource, user.data);
-          }
-        }}
-      >
-        {IconLookup(resource.iconName, themeContext.features.resources.icon.color)}
-        <ListItemContentLinkName>{resource.title}</ListItemContentLinkName>
-      </ListItemResourceLink>
-      <Checkbox
-        icon={<Icon icon={faHeart} />}
-        checkedIcon={<Icon icon={faHeart} color="#d73f09" />}
-        value={resource.id}
-        checked={favs}
-        onChange={handleChange}
-        inputProps={{
-          'aria-label': favoriteLabelText(favs),
-        }}
-      />
-    </ListItemFlex>
+    <>
+      <ListItemFlex>
+        <ListItemResourceLink
+          onClick={() => {
+            // if resource's IT system has an error, open the dialog box
+            if (itSystemError) {
+              open();
+            }
+            // else, open link
+            else {
+              window.open(resource.link);
+              close();
+            }
+            event();
+            if (!resource.excludeTrending) {
+              TrendingEvent(resource, user.data);
+            }
+          }}
+        >
+          {IconLookup(resource.iconName, themeContext.features.resources.icon.color)}
+          <ListItemContentLinkName>{resource.title}</ListItemContentLinkName>
+          {itSystemError && (
+            <Icon
+              fontSize={fontSize[18]}
+              icon={faExclamationCircleSolid}
+              color={themeContext.features.itStatus.item.icon.partialOutage}
+              style={{ marginLeft: '5px', fontSize: '18px!important' }}
+              data-testid="warning-icon"
+            />
+          )}
+        </ListItemResourceLink>
+
+        <Checkbox
+          icon={<Icon icon={faHeart} />}
+          checkedIcon={<Icon icon={faHeart} color="#d73f09" />}
```

Comment:
In a case like this, instead of the RGB color you'd want to use `Color['orange-400']` after importing the colors from theme-colors

Reply:
...I see this isn't a line you changed, but it'd be worthwhile cleaning it up if you can.